data-raw/1_Input/SWRuns_InputData_SoilLayers_v9.csv

```diff
@@ -1 +0,0 @@
-Label,SoilDepth_cm,depth_L1,depth_L2,depth_L3,depth_L4,depth_L5,depth_L6,depth_L7,depth_L8,depth_L9,depth_L10,depth_L11,depth_L12,depth_L13,depth_L14,depth_L15,depth_L16,depth_L17,depth_L18,depth_L19,depth_L20
```

Comment:
This file should exist in this location and not be deleted.

Reply:
This also applies to the file below (data-raw/1_Input/datafiles/SWRuns_InputData_soils_v12.csv) . For some reason, I cannot comment there.